index.js

```diff
@@ -0,0 +1,3 @@
+import './modules/dateTime.js';
+import './modules/Classes.js';
+import './modules/Events.js';
```

Comment:
- [ ] **[OPTIONAL]:** In the future, I would suggest that instead of importing the entire files, you import just the specific functions or variables that you need in the file. This way, you will improve tremendously the performance of your projects, especially if you have a large codebase! 🤓 🛠️